### PR TITLE
Support alignment with beautify

### LINKED ISSUE #1: Support alignment with beautify
Beautify should output:

```
| left   | center | right |
| :----- | :----: | ----: |
| foo    |  bar   |   0.0 |
| barbar | foofoo |  10.0 |
```

instead of:

```
| left   | center | right |
| :----- | :----: | ----: |
| foo    | bar    | 0.0   |
| barbar | foofoo | 10.0  |
```

See brrd/Abricotine#146

### PR DESCRIPTION
**Pull request by FireFragment**
Beautify now outputs:
```
| left   | center | right |
| :----- | :----: | ----: |
| foo    |  bar   |   0.0 |
| barbar | foofoo |  10.0 |
```

instead of:

```
| left   | center | right |
| :----- | :----: | ----: |
| foo    | bar    | 0.0   |
| barbar | foofoo | 10.0  |
```

Closes #1 (and also [this issue from Abricotine](https://github.com/brrd/abricotine/issues/146))

## Add Table.getAlignment method

The PR also adds Table.getAlignment method, which returns alignment of any column. Alignment is determined by colons in the second line of the table.
The new beautify method uses this function, but it may be useful also in other scenarios.

### CODE REVIEW

File: src/celldown.coffee [hunk 1]
```diff
@@ -242,7 +254,14 @@ celldown = do () ->
                         cell = firstChar + ("-" for [1..size-2]).join("") + lastChar
                     else
                         missingChars = size - cell.length
-                        if missingChars > 0 then cell += (" " for [1..missingChars]).join("")
+                        # Used only if alignment isn't center
+                        spaces = if missingChars > 0 then (" " for [1..missingChars]).join("") else ""
+                        console.log @arr[rowIndex][colIndex]
```
Comment (brrd) [inline src/celldown.coffee]: Could you please remove this console.log?

File: src/celldown.coffee [hunk 2]
```diff
@@ -208,6 +208,18 @@ celldown = do () ->
             @arr[1][colIndex] = content
             return this
 
+        # Get 'colIndex'th column alignment. Undefined means, that there isn't specified any align (no colons at the second line).
+        getAlignment: (colIndex) ->
```
Comment (brrd) [inline src/celldown.coffee:212]: Could you write the documentation about this new method in README.md?

**Review by brrd — CHANGES_REQUESTED** [review]
Thank you very much for your contribution. It looks good, I only have two minor requests before merging. I'm available if you have any question.

File: src/celldown.coffee (hunk 1 above)
Comment (FireFragment) [reply src/celldown.coffee]: I'm sorry, I forgot to remove it.

**Comment by FireFragment** [comment]
I made changes you requested and also these two little tweaks:
 - `getAlignment` method now uses a column where is cursor when colIndex argument is not defined (like the `align` method)
 - Improved example of `beautify` method in README.md to show, how it aligns contents of the table.

**Review by brrd — APPROVED** [review]

**Comment by brrd** [comment]
This looks good. Thank you for those changes.

**Pull request merged**